queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-information-block.jinja2

```diff
@@ -0,0 +1,70 @@
+{% include './common/20px.html' %}
+{% include './common/divider.html' %}
+{% include './common/20px.html' %}
+<div class="business-information">
+  <div>
+    {% if business.legalType == 'CP' %}
+    <div class="name bold">Cooperative Association Name:</div>
+    {% elif business.legalType in ['SP', 'GP'] %}
+    <div class="name bold">Business Name:</div>
+    {% else %}
+    <div class="name bold">Company Name:</div>
+    {% endif %}
```

Comment:
I feel guilty about creating an almost exact duplicate of 'business-information.html`, but I didn't want to modify a template that other outputs use. If you like the use of jinja blocks, we can move all output to use files like this.

Reply:
I won't suggest that for now. Patrick is working in parallel to change the way we send emails (which is plain text). All the NR emails are changed (Here is an example: https://github.com/bcgov/lear/blob/main/queue_services/entity-emailer/src/entity_emailer/email_templates/NR-BEFORE-EXPIRY.html). Not sure by when we will be changing lear emails to that format.

Reply:
Understood